.github/workflows/cd.yaml

```diff
@@ -0,0 +1,52 @@
+name: Release
+on:
+  workflow_run:
+    workflows: [ ci ]
+    branches: [ master ]
+    types:
+      - completed
```

Comment:
this doesn't check if ci failed: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow